lib/xenopsd.ml

```diff
@@ -55,6 +55,7 @@ let options = [
     "default-vbd-backend-kind", Arg.Set_string default_vbd_backend_kind, (fun () -> !default_vbd_backend_kind), "Default backend for VBDs";
     "ca-140252-workaround", Arg.Bool (fun x -> ca_140252_workaround := x), (fun () -> string_of_bool !ca_140252_workaround), "Workaround for evtchn misalignment for legacy PV tools";
     "additional-ballooning-timeout", Arg.Set_float additional_ballooning_timeout, (fun () -> string_of_float !additional_ballooning_timeout), "Time we allow the guests to do additional memory ballooning before live migration";
+    "domain_shutdown_ack_timeout", Arg.Set_float Resources.domain_shutdown_ack_timeout, (fun () -> string_of_float !Resources.domain_shutdown_ack_timeout), "Time to wait for in-guest PV drivers to acknowledge a shutdown request before we conclude that the drivers have failed";
```

Comment:
Move this to resources.ml as well (keeping things together).

Reply:
Oh, there are no `options` there... and resources.ml seems to have just file system stuff in it. Perhaps it's better to just put the ref on top of xenops_server.ml, since that is where it is used.

Reply:
Updated